module/applications/actor/config/hit-points-config.mjs

```diff
@@ -0,0 +1,116 @@
+import { formatNumber } from "../../../utils.mjs";
+import BaseConfigSheet from "../api/base-config-sheet.mjs";
+
+/**
+ * Configuration application for hit point bonuses and current values.
+ */
+export default class HitPointsConfig extends BaseConfigSheet {
+  /** @override */
+  static DEFAULT_OPTIONS = {
+    classes: ["hit-points"],
+    actions: {
+      roll: HitPointsConfig.#rollFormula
+    },
+    position: {
+      width: 420
+    }
+  };
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  static PARTS = {
+    config: {
+      template: "systems/dnd5e/templates/actors/config/hit-points-config.hbs"
+    }
+  };
+
+  /* -------------------------------------------- */
+  /*  Properties                                  */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  get title() {
+    return game.i18n.localize("DND5E.HitPoints");
+  }
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _preparePartContext(partId, context, options) {
+    context = await super._preparePartContext(partId, context, options);
+    context.data = this.document.system.attributes.hp;
+    context.fields = this.document.system.schema.fields.attributes.fields.hp.fields;
+    context.source = this.document.system.toObject().attributes.hp;
+
+    // Display positive ability modifier as its own row, but if negative merge into classes totals
```

Comment:
Because you always gain at least 1 HP each level, regardless of how low your CON mod is, so there are cases where if the mod is negative then the math will be displayed incorrectly.

Reply:
I guess you could work out how much of their negative Con contributed to their HP total since you know all their HP rolls. But I agree that would be difficult to display in a way that isn't confusing.